File: serverv2/PublicAPIs/Entities/Article.cs
namespace VideoGameArchive.Entities
{
    public class Article
    {
        public Article() {}

        public string title { get; set; }
        public string subtitle { get; set; }
        public string url { get; set; }
        public string author { get; set; }
        public string website { get; set; }
        public int datePublished { get; set; }
        public string thumbnail { get; set; }
        public string articleType { get; set; }
        public bool isArchivied { get; set; }
    }
}

File: website/src/code/entities/Article.ts
import { CalendarDate } from "./CalendarDate"

export class Article {
    constructor() {
        this.date = new CalendarDate(1995, 1, 1);
        this.title = "";
        this.subtitle = "";
        this.author = "";
        this.url = "";
        this.website = "";
        this.thumbnail = "";
        this.type = "";
    }

    date : CalendarDate;
    title: string;
    subtitle: string;
    author: string;
    url: string;
    website: string;
    thumbnail: string;
    type: string;
}